### PR TITLE
Add SSHPASS workflow

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

**Comment by andrewssobral** [comment]
@nebilbenmabrouk @fviale two possible improvements for this workflow:

1. Test if `sshpass` exists before running docker command.
```
if ! command -v sshpass &> /dev/null
then
    echo "sshpass could not be found"
    # run docker here...
else
    echo "sshpass found"
    # run sshpass here...
fi
```

2. Put the script to be executed in the remote machine as a prescript and use the generic info `PRE_SCRIPT_AS_FILE`.


What do you think?

**Comment by nebilbenmabrouk** [comment]
> @nebilbenmabrouk @fviale two possible improvements for this workflow:
> 
> 1. Test if `sshpass` exists before running docker command.
> 
> ```
> if ! command -v sshpass &> /dev/null
> then
>     echo "sshpass could not be found"
>     # run docker here...
> else
>     echo "sshpass found"
>     # run sshpass here...
> fi
> ```
> 
> 1. Put the script to be executed in the remote machine as a prescript and use the generic info `PRE_SCRIPT_AS_FILE`.
> 
> What do you think?

Thanks Andrews for the suggestions. OK I'll do that.

File: GetStarted/resources/catalog/SSH_Remote_Script_Execution.xml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        <script>
+          <code language="bash">
+            <![CDATA[
+# Get the SSH user and password to log into the remote machine
+sshUser=$variables_SSH_USR
+sshPassword=$variables_SSH_PWD
+remoteMachine=$variables_REMOTE_MACHINE
+
+# Prepare the script to execute in the remote machine
+mkdir -p scripts
```
[the remark this answers is not in the document]
Comment (nebilbenmabrouk) [reply GetStarted/resources/catalog/SSH_Remote_Script_Execution.xml]: Done

**Review by andrewssobral — APPROVED** [review]
Nice work @nebilbenmabrouk !

File: GetStarted/resources/catalog/SSH_Remote_Script_Execution.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+echo "Executing the script remotely in the machine "$remoteMachine
+
+if ! command -v sshpass &> /dev/null
+then
+    echo "sshpass could not be found, using a sshpass docker container"
+    
+    # copy the script to the remote machine
+    docker run --rm -i -v $(pwd)/scripts:/scripts ictu/sshpass -p $sshPassword scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no /scripts/script.sh $sshUser@$remoteMachine:/tmp/script.sh
+    
+    sleep 5
```
Comment (fviale) [inline GetStarted/resources/catalog/SSH_Remote_Script_Execution.xml:67]: why this sleep ?

**Review by fviale — APPROVED** [review]

**Pull request merged**